discovery_engine_core/web-api/src/db.rs

```diff
@@ -53,22 +53,22 @@ impl AppState {
 }
 
 #[derive(Clone, Debug)]
-pub(crate) struct InitConfig {
+pub struct InitConfig {
     /// S-mBert vocabulary path.
-    pub(crate) smbert_vocab: PathBuf,
+    pub smbert_vocab: PathBuf,
     /// S-mBert model path.
-    pub(crate) smbert_model: PathBuf,
-    /// List of [IngestedDocument]s in JSON format.
-    pub(crate) data_store: PathBuf,
+    pub smbert_model: PathBuf,
+    /// List of IngestedDocuments in JSON format.
```

Comment:
```suggestion
    /// List of `IngestedDocument`s in JSON format.
```

Reply:
This will be removed anyways in my PR #610 https://github.com/xaynetwork/xayn_discovery_engine/pull/610/files#diff-f66117a27820eb78841f8c365d078ab2a1817d6cbf5a03d26728c5e58627faf7L61